frontend/services/application/example/exampleApplicationService.ts

```diff
@@ -31,6 +31,29 @@ export class ExampleApplicationService {
     return await this.list(projectId, options)
   }
 
+  public async fetchAll(
+    projectId: string,
+    q: string,
+    isChecked: string
+  ): Promise<ExampleListDTO> {
+    const options: SearchOption = {
+      limit: '10000',
```

Comment:
I did something similar to this, but instead of naming the function fetchAll, i named the function `fetchByLimit` and get the limit from the previously called `fetchOne function`. There is the `count` given in the api and we can use that to set our limit XD    this would be useful in case our data surpassed 10000

Reply:
Oh right, we can set the limit based on `count` of the first api response. I will change it too XD